ui/src/components/home/NewHereSection.js

```diff
@@ -58,7 +58,7 @@ const NewHereSection = () => {
     <>
       <Flex
         w="full"
-        h={["100vh", "100vh", "100vh", "100vh", "80vh"]}
+        h="100vh"
```

Comment:
@Midnite1212 nope. opening this on a desktop makes it really empty. There's a reason why it was 80vh for the highest breakpoint

Reply:
![Screenshot 2021-09-19 at 4 50 23 PM](https://user-images.githubusercontent.com/33679483/133921374-5dade404-9d98-43fd-834a-cd448c7249f9.png)